agent/brandon/agent.py

```diff
@@ -0,0 +1,34 @@
+from threading import Thread
+
+from agent.base import BaseAgent
+from agent.brandon.search import Search
+from core.board import Board
+from core.color import Color
+from ui.model.heuristic_type import HeuristicType
+
+
+def agent_thread(search, board, color, on_find, on_complete):
+    search.start(board, color, on_find)
+    on_complete()
```

Comment:
We can discuss this sometime but the callback interface we're using right now doesn't work for multiprocessing since lambdas and process subclasses aren't serializable for whatever reason. The workaround I've been using has just been a simple generator setup; the downside is that the operator has to either hook into the main update loop (ugly) or spawn a new update thread specifically to poll the generator every frame (slow).